### PR TITLE
[PartDesign] Linting and cleaning superfluous whitespace

### PR DESCRIPTION
**Pull request by luzpaz**
Addressing .py files in `Mod/PartDesign/Scripts` and `Mod/PartDesign/WizardShaft`

### CODE REVIEW

File: src/Mod/PartDesign/Scripts/Spring.py [hunk 1]
```diff
@@ -9,10 +9,10 @@
 class MySpring:
    def __init__(self, obj):
       ''' Add the properties: Pitch, Diameter, Height, BarDiameter '''
-      obj.addProperty("App::PropertyLength","Pitch","MySpring","Pitch of the helix").Pitch=5.0
-      obj.addProperty("App::PropertyLength","Diameter","MySpring","Diameter of the helix").Diameter=6.0
-      obj.addProperty("App::PropertyLength","Height","MySpring","Height of the helix").Height=30.0
-      obj.addProperty("App::PropertyLength","BarDiameter","MySpring","Diameter of the bar").BarDiameter=3.0
+      obj.addProperty("App::PropertyLength", "Pitch", "MySpring", "Pitch of the helix").Pitch=5.0
```
Comment (chennes) [inline src/Mod/PartDesign/Scripts/Spring.py:12]: I'm surprised the linter didn't want spaces around that final "=" at the end of this line and the next few.

**Review by chennes — APPROVED** [review]
All of these changes should be harmless: this looks good to go.

File: src/Mod/PartDesign/Scripts/Spring.py (hunk 1 above)
Comment (AjinkyaDahale) [reply src/Mod/PartDesign/Scripts/Spring.py:12]: I'm guessing here, but maybe the linter just saw the parentheses opening for a method and didn't expect them to close before the line's end.

Comment (luzpaz) [reply src/Mod/PartDesign/Scripts/Spring.py]: Sorry, missed that feedback chennes. I've added it and rebased in 1b488ce

**Comment by luzpaz** [comment]
Rebased.  
@kkremitzki this is ready for merge

**Pull request merged**